auth/downscoping/README.md

```diff
@@ -0,0 +1,33 @@
+# Downscoping with Credential Access Boundaries
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=auth/downscoping/README.md
+
+This section contains samples for
+[Downscoping with Credential Access Boundaries](https://cloud.google.com/iam/docs/downscoping-short-lived-credentials).
+
+## Running the samples
+
+1. Your environment must be setup with [authentication
+information](https://developers.google.com/identity/protocols/application-default-credentials#howtheywork). If you're running on Compute Engine, this is already setup. You can also use `gcloud auth application-default login`.
```

Comment:
Can compute engine credentials be the source credentials?

Reply:
Yeah they should work. These are basically service account tokens and they can be downscoped.